server/server/src/lib.rs

```diff
@@ -68,10 +68,10 @@ pub fn verify_client_version<Req: Request>(
     request: &RequestWrapper<Req>,
 ) -> Result<(), ErrorWrapper<Req::Error>> {
     match &request.client_version as &str {
-        "0.3.17" => Ok(()),
-        "0.4.0" => Ok(()),
+        "0.3.17" => Err(ErrorWrapper::<Req::Error>::ClientUpdateRequired),
+        "0.4.0" => Err(ErrorWrapper::<Req::Error>::ClientUpdateRequired),
         "0.4.1" => Err(ErrorWrapper::<Req::Error>::ClientUpdateRequired),
-        "0.4.2" => Ok(()),
+        "0.4.2" => Err(ErrorWrapper::<Req::Error>::ClientUpdateRequired),
```

Comment:
Lol I was more so thinking we should allow 0.4.2 and then let the underscore handle the rest

Reply:
Good point, my brain was on auto-pilot I guess. 😂